### PR TITLE
feat: add file uploader functionality

### PR DESCRIPTION
**Pull request by sabhas**

### CODE REVIEW

File: src/components/file-uploader.component.tsx [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+import TableBody from '@material-ui/core/TableBody'
+import TableCell from '@material-ui/core/TableCell'
+import TableContainer from '@material-ui/core/TableContainer'
+import TableHead from '@material-ui/core/TableHead'
+import TableRow from '@material-ui/core/TableRow'
+import Paper from '@material-ui/core/Paper'
+
+import { SASContext } from '../context/sasContext'
+
+const useStyles = makeStyles((theme) => ({
```
Comment (YuryShkoda) [inline src/components/file-uploader.component.tsx:17]: `theme` attribute is not used

File: src/components/file-uploader.component.tsx [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+      setUploadDisabled(false)
+      setLocation(`${uploadPath}/${file.name}`)
+      setFileSize(bytesToSize(file.size))
+    }
+    setDirList([])
+    setErrorMessage('')
+  }, [file, uploadPath])
+
+  const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    switch (event.target.id) {
```
Comment (YuryShkoda) [inline src/components/file-uploader.component.tsx:65]: would be nice to add `default` block

File: src/components/file-uploader.component.tsx [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+        break
+    }
+  }
+
+  const handleUpload = async () => {
+    setUploadDisabled(true)
+    setIsUploading(true)
+    setDirList([])
+    setErrorMessage('')
+    console.log('upload button clicked')
```
Comment (YuryShkoda) [inline src/components/file-uploader.component.tsx:82]: is this `console.log` needed?

**Review by YuryShkoda — CHANGES_REQUESTED** [review]
see comments above

**Review by YuryShkoda — APPROVED** [review]

**Review by allanbowe — APPROVED** [review]

**Pull request merged**